src/components/KMultiselect/KMultiselect.vue

```diff
@@ -323,22 +339,22 @@ const props = defineProps({
    */
   width: {
     type: String,
-    default: '',
+    default: '100%',
   },
   /**
    * Number of rows of selections to show when focused
    */
   selectedRowCount: {
     type: Number,
-    default: 2,
+    default: 1,
   },
   /**
    * Determines whether to show total selected count (false), or
    * row(s) of selections when collapsed
    */
   collapsedContext: {
     type: Boolean,
-    default: false,
+    default: true,
```

Comment:
question: why is the default changing?

Reply:
Figma designs suggest that expended appearance is gonna be the default